### DISCUSSION TITLE
How long does it take to get approved for github sponsorship?

### DISCUSSION THREAD

**Discussion by Ndacyayisenga-droid** [community/community#60049] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

I applied for github sposorship almost a month ago but have never recieved feedback as yet. i would like to know approximately how long it takes
<img width="739" alt="Screenshot 1402-04-13 at 13 56 40" src="https://github.com/community/community/assets/58124613/4ba55742-a41d-43a1-839e-4be7113a1ad6">

**Comment by DSDmark** [comment]
Hi @Ndacyayisenga-droid,

> It takes GitHub a few business days to a week to review and approve your application for GitHub sponsorship. The approval process for GitHub Sponsors can vary depending on a number of factors. Typically, This is because the GitHub team needs to review your application to ensure it meets their guidelines. They may need to verify your identity, check your project's relevance, and ensure that it adheres to their terms of service.

For more info, you can read here: [**GitHub docs**](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account)

**Reply by Ndacyayisenga-droid** [reply]
Okay.
Just not certain why mine has taken this long. 
Note: Mine is a personal account

**Comment by anjuan** [comment]
Hi @Ndacyayisenga-droid,

I'm the Engineering Manager for GitHub Sponsors. I understand how it can be hard to submit your application for the program and wait for it to be approved. It can take our staff some time to review your application and process it. If approved, you can start setting up your profile. You can read more about the process [here](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/setting-up-github-sponsors-for-your-personal-account).

**Comment by moonfury-ops** [comment]
Thank you

**Comment by KernelBornie** [comment]
My application is almost approaching 2 months since I applied

**Comment by acagliano** [comment]
Mine is 2-3 weeks pending.
